Dockerfile.ci

```diff
@@ -1,12 +1,12 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.18 AS builder
```

Comment:
I am not sure if we are running ci using  rhel images or now . Any comment on this ? Do we have CI integrated for upstream now